purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -77,24 +77,32 @@ open class PurchasesConfiguration(builder: Builder) {
          */
         fun diagnosticsEnabled(diagnosticsEnabled: Boolean) = apply {
             this.diagnosticsEnabled = diagnosticsEnabled
+            if (!diagnosticsEnabled) {
+                this.verificationMode = EntitlementVerificationMode.DISABLED
+            }
```

Comment:
This seems weird. I understand doing that in the other mode, but why disable it here when turning off diagnostics?

Reply:
Well, users could get around the limitation if they first enable entitlement verification, and then disable diagnostics, since this is following a builder pattern. That way, they would get the feature without enabling diagnostics. I can see that this could be confusing though, and it's not a big deal, so maybe we are ok with allowing users to bypass the system like that?

Reply:
In the end I removed this as discussed over Slack